apps/dashr-financial-report/assets/FinancialFunctions.R

```diff
@@ -0,0 +1,122 @@
+
+
+# Generate a Table Function 
+generate_table <- function(df, nrows=20)
+  
+  # function generates a dash table from a supplied data frame (df)

```

Comment:
Should maybe update this comment to replace `dash table` with just `table` to avoid confusion, since this is giving an `htmlTable` rather than a `dashTable`

Reply:
Fixed, replaced with htmlTable.